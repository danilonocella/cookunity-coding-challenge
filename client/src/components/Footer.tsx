import { FC } from "react";
import { Typography, Box, Grid } from "@mui/material";

const Footer: FC = () => (
  <Box
    component="footer"
    sx={{
      py: 1,
      px: 2,
      mt: "auto",
      backgroundColor: (theme) => theme.palette.primary.main,
    }}
  >
    <Grid
      container
      direction="row"
      justifyContent="center"
      alignItems="center"
      sx={{padding: 2}}
    >
      <Grid item>
        <Typography
          variant="body2"
          color={(theme) => theme.palette.primary.contrastText}
        >
          {`Engineer Coding Challenge - Copyright Â© ${new Date().getFullYear()} CookUnity`}
        </Typography>
      </Grid>
    </Grid>
  </Box>
);

export default Footer;
