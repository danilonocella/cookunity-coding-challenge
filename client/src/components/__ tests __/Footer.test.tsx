import { render } from "@testing-library/react";
import Footer from "../Footer";

// Snapshot test for Footer component
test("Footer component matches snapshot", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
