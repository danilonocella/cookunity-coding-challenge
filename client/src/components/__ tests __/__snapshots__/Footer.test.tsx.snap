// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<div>
  <footer
    class="MuiBox-root css-1v6jfi3"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-1wxd26l-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1cl5n3e-MuiTypography-root"
        >
          Engineer Coding Challenge - Copyright Â© 2024 CookUnity
        </p>
      </div>
    </div>
  </footer>
</div>
`;
